modulemd/modulemd-module-index.c

```diff
@@ -117,6 +118,7 @@ modulemd_module_index_init (ModulemdModuleIndex *self)
 {
   self->modules =
     g_hash_table_new_full (g_str_hash, g_str_equal, g_free, g_object_unref);
+  self->stream_mdversion = modulemd_get_default_stream_mdversion ();
```

Comment:
@sgallagher After our discussions today, it may be best to leave this unset (`MD_DEFAULTS_VERSION_UNSET`)--at least until we work out how to deal with the bigger stream upgrade problem.

Reply:
Yeah, leave it unset for now.

Reply:
I reverted that change.